csrc/src/bias_act.cpp

```diff
@@ -101,18 +101,18 @@ STYLEGANR_API void * c_styleganr_bias_act (void* x, void* b, void* xref, void* y
 {
     //LANTERN_FUNCTION_START
     torch::Tensor result = bias_act(
-        reinterpret_cast<LanternObject<torch::Tensor>*>(x)->get(), 
-        reinterpret_cast<LanternObject<torch::Tensor>*>(b)->get(), 
-        reinterpret_cast<LanternObject<torch::Tensor>*>(xref)->get(), 
-        reinterpret_cast<LanternObject<torch::Tensor>*>(yref)->get(), 
-        reinterpret_cast<LanternObject<torch::Tensor>*>(dy)->get(), 
+        from_raw::Tensor(x), 
```

Comment:
Looks good but need to remove `#include "lantern_ob.h"` from top of file and replace with `#include <lantern/types.h>`